src/api/app.py

```diff
@@ -205,6 +207,24 @@ def log_out():
 def add_user_event():
     return event_controller.add_event(json.loads(request.data))
 
+@app.route('/api/addcourse', methods=['POST'])
+def add_student_course():
+    info = request.get_json()
+    resp, error = course_select.add_selection(info['name'], info['semester'], info['uid'], info['cid'])
+    return Response(status=200) if not error else Response(error, status=500)
+
+
+@app.route('/api/removecourse', methods=['POST'])
+def remove_student_course():
+    info = request.get_json()
+    resp, error = course_select.remove_selection(info['name'], info['semester'], info['uid'], info['cid'])
+    return Response(status=200) if not error else Response(error, status=500)
```

Comment:
reason for not doing methods=['DELETE']? 🤔

Reply:
Can you pass an argument to 'DELETE'?

Reply:
Yes.

https://github.com/YACS-RCOS/yacs.n/blob/clean_semester_upload_and_preview/src/web/src/services/UserService.js#L8-L15

Reply:
Done